mlflow/transformers/__init__.py

```diff
@@ -548,20 +548,19 @@ def save_model(
             default_reqs = sorted(set(inferred_reqs).union(default_reqs))
         else:
             default_reqs = None
+            if duplicates := _find_duplicate_requirements(pip_requirements):
+                _logger.warning(
+                    "Duplicate packages are present within the pip requirements. "
+                    f"Duplicate packages: {duplicates}. Please manually specify "
+                    "the requirements by using the `pip_requirements` argument in "
+                    "order to prevent unexpected installation issues for this model."
```

Comment:
```suggestion
                    "Duplicate packages are present within the specified `pip_requirements`. "
                    f"Duplicate packages: {duplicates}. Please ensure that it has no duplicated "
                    "packages in order to prevent unexpected installation issues for this model."
```

Is it possible that this function is called with non-null `pip_requirements` though user doesn't specify it? If not, the error message could be more specific.